app/config/constants.js

```diff
@@ -8,5 +8,6 @@ export default {
   },
   PUBLICATION_CHANNEL: {
     MAILING_LIST: 'http://themis.vlaanderen.be/id/publicatiekanaal/c184f026-feaa-4899-ba06-fd3a03df599c',
+    SUBSCRIBERS_FLANDERS_BE: 'http://themis.vlaanderen.be/id/publicatiekanaal/c06c30f5-d9e9-490e-822c-43faa9d0b25e'
```

Comment:
Hier is precies wat verwarring rond de verschillende publicatiekanalen. Er is:
- abonnees Vlaanderen.be (`SUBSCRIBERS_FLANDERS_BE`)
- website Vlaanderen.be (`WEBSITE_FLANDERS_BE`)

Het is `WEBSITE_FLANDERS_BE` dat automatisch toegevoegd moet worden als een van de 2 anderen geselecteerd wordt.

Voor de URIs: zie de [migration in de backend](https://github.com/kanselarij-vlaanderen/app-persberichten/blob/master/config/migrations/20210510100000-publication-channels.sparql#L14-L20)